### PR TITLE
fix: deployment run through fixes

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #

### CODE REVIEW

**Review by willmcvay — APPROVED** [review]
Just a few small things, happy for you to merge though

File: packages/cli/src/commands/react-starter.ts [hunk 1]
```diff
@@ -36,6 +43,30 @@ export class ReactStarterCommand extends AbstractCommand {
     })
   }
 
+  protected async installServerlessDeps(path: string): Promise<void> {
+    const deps = [
+      'serverless-deployment-bucket',
```
[the remark this answers is not in the document]
Comment (bashleigh) [reply packages/cli/src/commands/react-starter.ts:48]: yea this is a bit of a mess at the moment

File: packages/cli/src/abstract.command.ts [hunk 2]
```diff
@@ -71,7 +71,7 @@ export abstract class AbstractCommand {
     }
 
     const instance = axios.create({
-      baseURL: config ? config.config.baseUrl : 'https://developer.reapit.com/',
+      baseURL: config?.config?.baseUrl || 'https://h2r8e8wbd4.execute-api.eu-west-2.amazonaws.com/dev/',
```
[the remark this answers is not in the document]
Comment (bashleigh) [reply packages/cli/src/abstract.command.ts:74]: yea I need to sort this properly :)

File: packages/cli/src/commands/react-starter.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (bashleigh) [reply packages/cli/src/commands/react-starter.ts:48]: Yes please.

File: packages/cli/src/abstract.command.ts [hunk 3]
```diff
@@ -103,25 +105,61 @@ export abstract class AbstractCommand {
...
+              .filter((arg) => arg.type === 'option')
+              .map((arg) => `[--${arg.name}${arg.shortName ? `|-${arg.shortName}` : ''}]`)
+              .join(' ')
+      }`,
+      parent ? 2 : 1,
+      '  ',
+    )
+    this.writeLine('')
+
+    // console.log(`
```
Comment (willmcvay) [inline packages/cli/src/abstract.command.ts]: Do we need all this now?

**Review by willmcvay — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**